Makefile

```diff
@@ -166,33 +166,35 @@ bin/subctl-%: generate-embeddedyamls $(shell find pkg/subctl/ -name "*.go") vend
 
 ci: generate-embeddedyamls golangci-lint markdownlint unit build images
 
-generate-embeddedyamls: generate pkg/subctl/operator/common/embeddedyamls/yamls.go
+generate-embeddedyamls: pkg/subctl/operator/common/embeddedyamls/yamls.go
```

Comment:
Can't this target be dropped then? It's only used in operator and I doubt anyone is running it manually.

Also this line is the same as 171.